profiles/salvinanto7-15042000.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hacktober Fest 2020 - Contributor Page</title>
+    <link rel="stylesheet" href="card.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <div class="event-title">
+        <h1>Hacktober Fest 2020 - Contributor Page</h1>
+    </div>
+    <section class="card-list">
+
+        <!-------------------------------------------Copy code from below------------------------------------------------>
+
+        <div class="yourname-dob">
```

Comment:
da can you change the div with the class "yourname-dob" to your name and dob. For example, class="salvinanto15042000"
This is to keep it random while adding all of this to the same file later.

Reply:
oh.. ok